app/controllers/notifications_controller.rb

```diff
@@ -4,20 +4,24 @@ class NotificationsController < ApplicationController
   before_action :render_home_page_unless_authenticated, only: [:index]
 
   def index
-    scope = current_user.notifications
-    scope = params[:archive].present? ? scope.archived : scope.inbox
+    notifications = current_user.notifications
```

Comment:
Technically this is still a scope because ActiveRecord doesn't query the data until we ask it to iterate over or return the list of objects. So I don't quite agree with this name change.